scalecube-organization/src/main/java/io/scalecube/organization/operation/GetOrganizationMembers.java

```diff
@@ -21,11 +21,9 @@ protected GetOrganizationMembersResponse process(
     Organization organization = getOrganization(request.organizationId());
 
     checkSuperUserAccess(organization, context.profile());
-    Collection<OrganizationMember> organizationMembers =
-        context.repository().getOrganizationMembers(organization);
-    OrganizationMember[] members = new OrganizationMember[organizationMembers.size()];
 
-    return new GetOrganizationMembersResponse(organizationMembers.toArray(members));
+    return new GetOrganizationMembersResponse(
+        organization.members().toArray(new OrganizationMember[0]));
```

Comment:
why did you set the array size to `0`?

Reply:
https://shipilev.net/blog/2016/arrays-wisdom-ancients/